<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindowui</class>
 <widget class="QWidget" name="loginwindowui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudySpace</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius : 10px;</string>
  </property>
  <widget class="QLineEdit" name="numberInput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="numberLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;账号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;
background-color : rgb(231, 231, 231);
font-size:24px;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(233, 248, 255)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Study Space&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainBackgroundLabel">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>95</y>
     <width>431</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(238, 238, 238);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="minimizeButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 24px;</string>
   </property>
   <property name="text">
    <string>—</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 24px;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 36px;
color: rgb(162, 162, 162);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moreinfoButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 36px;
color: rgb(162, 162, 162);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>更多</string>
   </property>
  </widget>
  <zorder>mainBackgroundLabel</zorder>
  <zorder>numberInput</zorder>
  <zorder>passwordInput</zorder>
  <zorder>numberLabel</zorder>
  <zorder>passwordLabel</zorder>
  <zorder>loginButton</zorder>
  <zorder>titleLabel</zorder>
  <zorder>minimizeButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>registerButton</zorder>
  <zorder>moreinfoButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
